packages/components/pie-webc-core/src/functions/wrapNativeEvent.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Wraps the native event in a custom event. This is useful for native events that have `composed` set to `false`
+ * as these cannot propagate outside of the shadow DOM.
```

Comment:
Can be done in follow up, but can you add the link to the docs/article you showed on this (or to our wiki/readme section if it's in there)